pkg/controllers/cstorvolumeconfig/controller.go

```diff
@@ -156,6 +169,25 @@ func (c *CVCController) syncCVC(cvc *apis.CStorVolumeConfig) error {
 	}
 
 	var err error
+
+	cvc, err = c.reconcileVersion(cvc)
```

Comment:
Before calling reconcileVersion, we need to populate CVC version details.

Reply:
Version will be populated while creation itself for CVC.